### PR TITLE
AWS SNS Channel Launch

### PR DESCRIPTION
**Pull request by claudsonm**
Sending the files already with the changes pointed in the code review https://github.com/laravel-notification-channels/channels/pull/81.

### CODE REVIEW

File: tests/SnsServiceProviderTest.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+use Aws\Sns\SnsClient as SnsService;
+use Illuminate\Contracts\Foundation\Application;
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
+use NotificationChannels\AwsSns\Sns;
+use NotificationChannels\AwsSns\SnsChannel;
+use NotificationChannels\AwsSns\SnsServiceProvider;
+
+class SnsServiceProviderTest extends MockeryTestCase
```
Comment (atymic) [inline tests/SnsServiceProviderTest.php]: Try extending the default `TestCase` and using the mockery trait `use MockeryPHPUnitIntegration;`

**Comment by atymic** [comment]
Looks like that didn't work. Maybe just drop the `lowest` dep runs from the travis build

**Comment by claudsonm** [comment]
Yep. I was giving a try but I'm going to do that. Just finishing up some style ci checks.

**Comment by atymic** [comment]
MWR 🎉

**Pull request merged**